build/make-certs.py

```diff
@@ -10,18 +10,18 @@
 
 class CockroachCluster(object):
 
-    def __init__(self, cluster, ca_cert_to_join=None):
+    def __init__(self, cluster, ca_cert_to_join=None, namespace=None):
```

Comment:
It seems like we never omit the `namespace` argument and a value of None doesn't really make sense when constructing `node_addresses`; let's just make this argument required with no default value.

Reply:
Done.